Index.js fails on npm run download "Cannot find module 'lodash'"

Some quick specs:
npm version 6.9.0 
node version 10.16.3
OS: Windows 10

As the title states, running the npm run download command fails with the following errors: 

Error: Cannot find module 'lodash'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (C:\users\user\repos\n64\libdragon\index.js:5:11)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! libdragon@1.3.5 tc: `node index.js "download"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the libdragon@1.3.5 tc script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

The log file mentioned in the trace contains the following:

0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'tc',
1 verbose cli   '--',
1 verbose cli   'download' ]
2 info using npm@6.9.0
3 info using node@v10.16.3
4 verbose run-script [ 'pretc', 'tc', 'posttc' ]
5 info lifecycle libdragon@1.3.5~pretc: libdragon@1.3.5
6 info lifecycle libdragon@1.3.5~tc: libdragon@1.3.5
7 verbose lifecycle libdragon@1.3.5~tc: unsafe-perm in lifecycle true
8 verbose lifecycle libdragon@1.3.5~tc: PATH: C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;C:\users\userl\repos\n64\libdragon\node_modules\.bin;C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;C:\users\user\repos\n64\libdragon\node_modules\.bin;C:\Users\user\bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\local\bin;C:\Program Files\Git\usr\bin;C:\Program Files\Git\usr\bin;C:\Program Files\Docker Toolbox;C:\Users\user\bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\local\bin;C:\Program Files\Git\usr\bin;C:\Program Files\Git\usr\bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Users\user\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.0\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.0\libnvvp;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\libnvvp;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\iCLS;C:\Program Files\Intel\Intel(R) Management Engine Components\iCLS;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\WiFi\bin;C:\Program Files\Common Files\Intel\WirelessCommon;C:\Python;C:\Python\Scripts;C:\Python\Lib\Site-Packages;C:\Program Files\Git\cmd;C:\Program Files\dotnet;C:\Program Files (x86)\GnuWin32\bin;C:\sqlite;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files\PuTTY;C:\PostgreSQL\pg11\bin;C:\Users\user\.cargo\bin;C:\Program Files\nodejs;C:\Users\user\.cargo\bin;C:\Users\user\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Docker Toolbox;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.0\lib\x64;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.0\extras\CUPTI\libx64;C:\Users\user\Anaconda3_64;C:\Users\user\Anaconda3;C:\Users\user\AppData\Roaming\npm;C:\Program Files\Git\usr\bin\vendor_perl;C:\Program Files\Git\usr\bin\core_perl;C:\Program Files\Git\usr\bin\vendor_perl;C:\Program Files\Git\usr\bin\core_perl
9 verbose lifecycle libdragon@1.3.5~tc: CWD: C:\users\user\repos\n64\libdragon
10 silly lifecycle libdragon@1.3.5~tc: Args: [ '/d /s /c', 'node index.js "download"' ]
11 silly lifecycle libdragon@1.3.5~tc: Returned: code: 1  signal: null
12 info lifecycle libdragon@1.3.5~tc: Failed to exec tc script
13 verbose stack Error: libdragon@1.3.5 tc: `node index.js "download"`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\index.js:301:16)
13 verbose stack     at EventEmitter.emit (events.js:198:13)
13 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:198:13)
13 verbose stack     at maybeClose (internal/child_process.js:982:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
14 verbose pkgid libdragon@1.3.5
15 verbose cwd C:\users\user\repos\n64\libdragon
16 verbose Windows_NT 10.0.17134
17 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "tc" "--" "download"
18 verbose node v10.16.3
19 verbose npm  v6.9.0
20 error code ELIFECYCLE
21 error errno 1
22 error libdragon@1.3.5 tc: `node index.js "download"`
22 error Exit status 1
23 error Failed at the libdragon@1.3.5 tc script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]

--- a/README.md
+++ b/README.md
@@ -6,6 +6,10 @@ To make managing the toolcain easier, a docker container is used. Node.js is use
 
 On a machine with node.js (>= 7.6) & docker you can simply do in this repository's root;
 
+    npm install
+
+to install all necessary dependencies and;
+
     npm run download
 
 to download the toolchain image from docker repository. If you instead prefer to build it on your computer do;

--- a/include/rsp.h
+++ b/include/rsp.h
@@ -13,6 +13,8 @@ extern "C" {
 void rsp_init();
 void load_ucode(void * start, unsigned long size);
 void read_ucode(void* start, unsigned long size);
+void load_data(void * start, unsigned long size);
+void read_data(void* start, unsigned long size);
 void run_ucode();
 
 #ifdef __cplusplus

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "libdragon",
-  "version": "1.3.7",
+  "version": "1.3.8",
   "description": "This is a simple library for N64 that allows one to code using the gcc compiler suite and nothing else.  No proprietary library is needed.  For more information, visit http://www.dragonminded.com/n64dev/",
   "main": "index.js",
   "engines": {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "libdragon",
-  "version": "1.3.5",
+  "version": "1.3.7",
   "description": "This is a simple library for N64 that allows one to code using the gcc compiler suite and nothing else.  No proprietary library is needed.  For more information, visit http://www.dragonminded.com/n64dev/",
   "main": "index.js",
   "engines": {

--- a/src/rsp.c
+++ b/src/rsp.c
@@ -82,6 +82,23 @@ void load_ucode(void* start, unsigned long size)
     return;
 }
 
+void load_data(void* start, unsigned long size)
+{
+    disable_interrupts();
+    __SP_DMA_wait();
+
+    SP_regs->DRAM_addr = start;
+    MEMORY_BARRIER();
+    SP_regs->RSP_addr = (void*)SP_DMA_DMEM;
+    MEMORY_BARRIER();
+    SP_regs->rsp_read_length = size - 1;
+    MEMORY_BARRIER();
+
+    __SP_DMA_wait();
+    enable_interrupts();
+    return;
+}
+
 void read_ucode(void* start, unsigned long size)
 {
     disable_interrupts();
@@ -99,6 +116,26 @@ void read_ucode(void* start, unsigned long size)
     enable_interrupts();
     return;
 }
+
+
+void read_data(void* start, unsigned long size)
+{
+    disable_interrupts();
+    __SP_DMA_wait();
+
+    SP_regs->DRAM_addr = start;
+    MEMORY_BARRIER();
+    SP_regs->RSP_addr = (void*)SP_DMA_DMEM;
+    MEMORY_BARRIER();
+    SP_regs->rsp_write_length = size - 1;
+    MEMORY_BARRIER();
+    __SP_DMA_wait();
+    data_cache_hit_invalidate(start, size);
+
+    enable_interrupts();
+    return;
+}
+
 
 void run_ucode()
 {

--- a/src/rsp.c
+++ b/src/rsp.c
@@ -67,8 +67,6 @@ void rsp_init()
 
 void load_ucode(void* start, unsigned long size)
 {
-    data_cache_hit_writeback_invalidate(start, size);
-
     disable_interrupts();
     __SP_DMA_wait();
 
@@ -81,15 +79,11 @@ void load_ucode(void* start, unsigned long size)
 
     __SP_DMA_wait();
     enable_interrupts();
-
-    data_cache_hit_invalidate(start, size);
     return;
 }
 
 void read_ucode(void* start, unsigned long size)
 {
-    data_cache_hit_writeback_invalidate(start, size);
-
     disable_interrupts();
     __SP_DMA_wait();
 

